paper-tabs.html

```diff
@@ -390,6 +420,62 @@
         }, 1);
       },
 
+      _activateHandler: function() {
```

Comment:
Could you add `@private` on the methods that are private? It will be closure compiler friendly ;)